platform/broadcom/sonic-platform-modules-accton/as7712-32x/cfg/th-as7712-32x100G.config.bcm

```diff
@@ -0,0 +1,295 @@
+# accton_as7710_32x 32x100G SDK config
```

Comment:
lguohan,
Thanks for the reminder~
We sent our SDK config to stcheng after we passed `config_checker` yesterday.